src/components/mods/modsTable.tsx

```diff
@@ -438,10 +478,37 @@ export const ModsTable = ({ qualities, difficulties, modsWithInfo, isLoading }:
 
 
 
+    const { cx, classes } = useStyles({ difficultyIndex: currentTabIndex });
+
+    const tabColors = [
+        classes.beginner,
+        classes.intermediate,
+        classes.advanced,
+        classes.expert,
+        classes.grandmaster
+    ];
 
-    const { cx, classes } = useStyles();
+    useEffect(() => {
```

Comment:
So the pagination dropdown menu DOM node lies outside the table, so (I think) we can't set it's style within createStyles. So I instead set the body's class (the classes are defined in global.css) whenever the tab changes.

This is a bad way to do this but I don't know if there is a better way - there is a way to color of the active selection in Mantine Datable, but I couldn't find anything for the entire dropdown.

Reply:
It would be a little better if we could use a [CSS Module](https://nextjs.org/docs/app/building-your-application/styling/css-modules) instead of global styles. I gave it a quick try in the [implement_color_palette_css_module](https://github.com/celestemods-com/CelesteMods/tree/implement_color_palette_css_module) branch, but I couldn't get it working in the time I had.

Please take a look and see if you can move this styling into a CSS Module.